### PR TITLE
[RSS-74] Style New Account Page - Desktop

### PR DESCRIPTION
**Pull request by ObjectJosh**
### Version Overview:
- Styled New Account page accordingly to desktop mockups

### Notes:
- `NewAccountPage`: Still need to find a way to get rid of the blue "focused/selection" background when the Show/Hide password button is clicked (I need you to help for this one b/c I can't figure it out)
- `NewAccountPage`: I added a min-width and max-width in addition to dynamic resizing, since I personally felt it looked better. Feel free to request edits if you'd like me to change it.

### Visual:
Screen Shot 2021-04-19 at 8 22 59 PM

### CODE REVIEW

**Comment by ObjectJosh** [comment]
Btw, I misremembered my number (RSS-74) and wrote (RSS-77), sorry!

File: src/components/NewAccountPage/NewAccountPage.jsx [hunk 1]
```diff
@@ -67,6 +107,10 @@ const StyledHeader = styled(Form.Label)`
    font-size: 36px;
    font-family: 'Nunito Sans', sans-serif;
    font-weight: 600;
+   @media only screen and (min-width: 768px) {
+      margin-left: auto;
+      margin-right: auto;
+   }
 `;
 
 const StyledHideButton = styled(Button)`
```
Comment (jackfales) [inline src/components/NewAccountPage/NewAccountPage.jsx:116]: Add the following to the StyledHideButton to fix the background color changing:
```
&:hover,
&:focus {
   background-color: transparent;
   color: #024e6b;
}
```

File: src/components/NewAccountPage/NewAccountPage.jsx [hunk 2]
```diff
@@ -84,8 +128,26 @@ const StyledButtonAndEye = styled.div`
...
-   margin-top: -125px;
+   margin-top: -60px;
    margin-right: 20px;
+   @media only screen and (min-width: 768px) {
+      margin-top: -6px;
+      position: absolute;
+      margin-left: auto;
+      margin-right: auto;
+   }
+`;
```
Comment (jackfales) [inline src/components/NewAccountPage/NewAccountPage.jsx]: Replace everything in this styled component with the following code. This fixes the weird spacing with the show/hide password button.

```
position: absolute;
top: 30px;
@media only screen and (max-width: 768px) {
   position: absolute;
   top: 30px;
   right: 40px;
}
```

**Review by jackfales — COMMENTED** [review]
Looks good! I added two comments to fix some minor things with the show/hide password button but after that it should be good to merge!

File: src/components/NewAccountPage/NewAccountPage.jsx (hunk 2 above)
Comment (ObjectJosh) [reply src/components/NewAccountPage/NewAccountPage.jsx:139]: It doesn't fix it on my screen. Maybe we need to come up with a solution that works for both our screen-testing sizes.

**Review by jackfales — APPROVED** [review]
Looks good

**Pull request merged**